bin/create-k8s-config-maps-and-secrets

```diff
@@ -38,6 +38,7 @@ data:
   OAUTH_REDIRECT_URI: "${OAUTH_REDIRECT_URI}"
   FALLBACK_AUTHN_ENABLED: "${FALLBACK_AUTHN_ENABLED}"
   CMR_ENDPOINT: "${CMR_ENDPOINT}"
+  CMR_MAX_PAGE_SIZE: "${CMR_MAX_PAGE_SIZE}"
```

Comment:
I'm a little worried about this being set as an environment variable in the query-cmr service. I thought each query-cmr work item is supposed to say how many granules to pull back in order to handle maxResults correctly and not pull back too many items.

Reply:
@vinnyinverso - can you review to figure out if the paging through items and limiting to maxResults still works the same?

Reply:
this is just used as the upper limit - the actual number used is set on the request
```
export async function queryGranulesWithSearchAfter(
  token: string,
  limit = cmrMaxPageSize,
  query?: CmrQuery,
  sessionKey?: string,
  searchAfterHeader?: string,
): Promise<CmrGranuleHits> {
  const baseQuery = {
    page_size: Math.min(limit, cmrMaxPageSize),
  };
  ```

Reply:
I can probably remove this if you like as I don't think we are ever don't pass in the limit.

Reply:
Done

Reply:
> @vinnyinverso - can you review to figure out if the paging through items and limiting to maxResults still works the same?

It looks like it'll be fine (also talked to James about it). Going to test it out locally now too.